core/trino-main/src/main/java/io/trino/Session.java

```diff
@@ -118,7 +120,8 @@ public Session(
             SessionPropertyManager sessionPropertyManager,
             Map<String, String> preparedStatements,
             ProtocolHeaders protocolHeaders,
-            Optional<Slice> exchangeEncryptionKey)
+            Optional<Slice> exchangeEncryptionKey,
+            Map<String, String> customTraceAttributes)
```

Comment:
What was the confusing place with `extra`?

Reply:
When I try to put myself in the place of an admin looking at some attribute for a query with a prefix "extra:..." I think what "extra" is referring to is ambiguous.  I don't think that it would be evident that "extra" is referring to the fact that it's an extra attribute in addition to the typical ones and that it could mean that the extra is in reference to the name of the attribute.  That might just be me though.